app/javascript/yoha-starter.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Yoha hand tracking with click capabilities.
+ * Adapted from official Yoha minimal JS hand-tracker example.
+ */
+import * as yoha from '@handtracking.io/yoha';
+import {
+  SetCursorColor,
+  SetCursorPosition,
+  SetCursorVisibility,
+  InitializeCursor,
+  SimulateClick,
+} from '@javascript/cursor';
+
+// eslint-disable-next-line import/prefer-default-export
+export async function Run() {
+  // Setup video feed.
+  const streamRes = await yoha.CreateMaxFpsMaxResStream();
+  if (streamRes.error) {
+    // browser denied or other error
+    // eslint-disable-next-line no-console
+    console.error(`[YOHA]: setup_video error: ${streamRes.error}`);
+    return;
+  }
+  const video = yoha.CreateVideoElementFromStream(streamRes.stream);
+
+  // Download models.
+  const modelFiles = await yoha.DownloadMultipleYohaTfjsModelBlobs(
+    '/assets/yoha/box/model.json',
+    '/assets/yoha/lan/model.json',
+    (rec, total) => {
+      // eslint-disable-next-line no-console
+      console.debug(`[YOHA]: Download progress: ${(rec / total) * 100}%`);
```

Comment:
how long does this download take?

Reply:
Not long at all, typically <1.5 seconds.